### PR TITLE
Revamp landing page design

### PR DESCRIPTION
**Pull request by eliabejr**
https://astro-landing.netlify.com/en/

- [x] general: form font color white
- [x] general: smaller font-size on mobile screens
- [x] home: missing the different purple colors on the text 
- [x] howwework: font-family are not ok on hover
- [x] howwework: maybe smaller last dot? @grsabreu 
- [x] opensource: fetch from githubapi our opensource? 
- [x] opensource: add star button + stars count
- [x] case: see figma
- [x] join us: add pictures of us?

### CODE REVIEW

File: src/Sections/OurStack.js [hunk 1]
```diff
@@ -1,95 +1,40 @@
...
+  display: flex;
+  padding-bottom: 50px;
+`
+
+const OurStack = () => (
+    <OurStackWrapper id="ourStack">
+      <Wrapper alignItems="center" direction="row" justifyContent="space-between" wrap="wrap">
+        <img src={ocamlLogo} alt="OCaml"/>
+        <img src={reasonLogo} alt="ReasonML"/>
+        <img src={jsLogo} alt="Javascript"/>
```
Comment (fakenickels) [inline src/Sections/OurStack.js]: minor casing fix: JavaScript

File: src/Sections/OurStack.js [hunk 2]
```diff
@@ -1,95 +1,40 @@
...
+        <img src={graphqlLogo} alt="GraphQl"/>
```
Comment (fakenickels) [inline src/Sections/OurStack.js]: minor casing fix: GraphQL

File: src/components/Logo.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  to {
+    transform: rotate(360deg);
+  }
+`
+
+const Image = styled.img`
+  width: 200px;
+  height: auto;
+  margin-bottom: 30px;
+  // animation: ${spin} 10s infinite linear;
```
Comment (fakenickels) [inline src/components/Logo.js]: if not used you can remove now

Comment (eliabejr) [reply src/components/Logo.js]: forgot to ask, what do you guys think about this? Logo should be spinning or no?
My vote: no

File: src/Sections/OurStack.js (hunk 1 above)
Comment (eliabejr) [reply src/Sections/OurStack.js]: fixed, thanks!

File: src/Sections/OurStack.js (hunk 2 above)
Comment (eliabejr) [reply src/Sections/OurStack.js]: fixed, thanks!

File: src/components/Logo.js (hunk 3 above)
Comment (guilhermedecampo) [reply src/components/Logo.js]: not anymore!

**Comment by eliabejr** [comment]
Deployed!
https://astro-landing.netlify.com/

**Review by fakenickels — APPROVED** [review]

**Comment by guilhermedecampo** [comment]
https://astro-landing.netlify.com/en/contact contact here is actually `join us`
We should have a new item in the menu to show the contact.. that's just the form we have in every page without the hire us.

**Comment by guilhermedecampo** [comment]
@eliabejr take a look on this: https://github.com/yahoo/react-intl/issues/68 maybe can be helpful on the <em>.

**Comment by eliabejr** [comment]
> https://astro-landing.netlify.com/en/contact contact here is actually `join us`
> We should have a new item in the menu to show the contact.. that's just the form we have in every page without the hire us.

Moved here! I think since we have that form in every page, we can simply put a anchor in navbar

**Pull request merged**